tests/cms/views/poi/test_poi_form.py

```diff
@@ -167,8 +167,8 @@ def test_poi_in_use_not_deleted(
             == f"{settings.LOGIN_URL}?next={delete_poi}"
         )
     elif role in HIGH_PRIV_STAFF_ROLES:
-        with pytest.raises(ProtectedError):
-            response = client.post(delete_poi)
```

Comment:
I think that is mostly fine. You can add an assertion that the poi with `poi_id` still exists after the post request, or check for the `messages.error` text if you add that